internal/api/mfa.go

```diff
@@ -343,6 +344,7 @@ func (a *API) challengePhoneFactor(w http.ResponseWriter, r *http.Request) error
 	}
 	return sendJSON(w, http.StatusOK, &ChallengeFactorResponse{
 		ID:        challenge.ID,
+		Type:      factor.FactorType,
```

Comment:
In enroll we call it `type` as well. Refraining from calling it `FactorType` in case we want to split the notion of `ChallengeType` from `FactorType`.